runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java

```diff
@@ -186,7 +187,7 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx,
         java.lang.Throwable cause) throws Exception {
-        log.error("exceptionCaught: Exception {} caught.", cause.getClass().getSimpleName(), cause);
+        log.error("exceptionCaught: Exception {} caught. Cause: {}", cause.getClass().getSimpleName(), cause);
```

Comment:
Does this make https://github.com/CorfuDB/CorfuDB/pull/1572/files#diff-74369a0020e71f6d710749351f0ba98eR475 redundant ?

Reply:
Not really right? the other code is on server logs.